OccupancyMap:
  PathToImage: $(find elevationmap_server)/config/sample.png
  GridResolution: 0.1
  FreeThreshold: 0.3
  OccupiedThreshold: 0.7